webrtc.html

```diff
@@ -7906,7 +7906,10 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             <dt><dfn><code>onerror</code></dfn> of type <span class=
             "idlAttrType"><a>EventHandler</a></span></dt>
             <dd>The event type of this event handler is <code><a data-for=
-            "RTCDataChannel">error</a></code>.</dd>
+            "RTCDataChannel">RTCErrorEvent</a></code>.
+            <code>RTCErrorEvent.errorDetail</code> contains the SCTP
+            Cause Code value, and <code>RTCErrorEvent.message</code>
+            contains the SCTP Cause-Specific-Information.</dd>
```

Comment:
Should this be ", possibly with additional text", so that it's clear that the UA can add stuff here, and it's not machine parseable? (My generic fear about "message": That people will start parsing it.)